tests/testthat/test-translate.r

```diff
@@ -146,6 +146,22 @@ test_that("str_trim() translates correctly ", {
       sql("LTRIM(RTRIM(\"field_name\"))"))
 })
 
+# lubridate -------------------------------------------
 
+test_that("as_date() translates correctly ", {
+  expect_equivalent(
+    translate_sql(
+      as_date("2017-01-01")
+    ),
+    sql("CAST('2017-01-01' AS DATE)")
+    )
+})
 
-
+test_that("as_datetime() translates correctly ", {
+  expect_equivalent(
+    translate_sql(
+      as_datetime("2017-01-01 01:02:03")
+    ),
+    sql("CAST('2017-01-01 01:02:03' AS DATETIME)")
```

Comment:
This test fails cause the translation on line 189 in `translate_sql_base.r` uses `TIMESTAMP` and not `DATETIME`